docs/ml/deploy/tflite_cpu.md

```diff
@@ -33,9 +33,9 @@ Enter a name or use the suggested name for your service and click **Create**.
 You can choose to configure your service with an existing model on the machine or deploy a model onto your machine:
 
 {{< tabs >}}
-{{% tab name="Deploy Model on Robot" %}}
+{{% tab name="Deploy Model on Machine" %}}
 
-1. To configure your service and deploy a model onto your machine, select **Deploy model on robot** for the **Deployment** field.
+1. To configure your service and deploy a model onto your machine, select **Deploy model on machine** for the **Deployment** field in the resulting vision service configuration pane,.
```

Comment:
Good catch-- idk why I never noticed robot-machine there 😭

Reply:
I changed the app to change this ;)

Reply:
Oh 🤣 Nice!